eZ/Publish/Core/Repository/RoleService.php

```diff
@@ -1072,6 +1139,31 @@ protected function validateRoleCreateStruct(APIRoleCreateStruct $roleCreateStruc
         return $allErrors;
     }
 
+    /**
+     * Validates Policies and Limitations in Role copy struct.
+     *
+     * @uses ::validatePolicy()
```

Comment:
This is not usable in the current format (at least for me). I can't find a reason we would need it. Let me know what was your intention here.

Reply:
@alongosz I kinda wanted to replicate the validation of `RoleCreateStruct`. As by your advice `RoleCopyStruct` now extends `RoleCreateStruct` I can safely use an existing method above and remove mentioned one: https://github.com/ezsystems/ezpublish-kernel/pull/3009/commits/ab92abaef38c3fc7bb95281600badd9eb16c501f